sites/friday/src/manage-resources/resource-init.md

```diff
@@ -13,42 +13,101 @@ keywords:
 
 | Resource type               | Amount |
 | --------------------------- | ----------- |
 | CPU                         | 0.5 |
 | RAM                         | Depends on the [container profile](/manage-resources/adjust-resources.md#advanced-container-profiles). |
 | Disk size (only applicable if the app or service requires a disk)                   | 512 MB |
 
-If you don't want to use those default resources, you can define a different [resource initialization strategy](/manage-resources/resource-init.md#define-a-resource-initialization-strategy).
-You can also [adjust those resources](/manage-resources/adjust-resources.md) after your project or new container has been deployed.
+If you don't want to use these default resources, you can set another [resource initialization strategy](/manage-resources/resource-init.md#set-a-resource-initialization-strategy).
+You can also [adjust resources](/manage-resources/adjust-resources.md) after your project or new container has been deployed.
+
+{{% note %}}
 
 For information on costs related to resource usage, see the [{{% vendor/name %}} pricing page](https://upsun.com/pricing/).
 Note that you can [keep an eye on those costs](/manage-resources/resource-billing.md) in the Console.
 
-## Define a resource initialization strategy
+{{% /note %}}
 
-You can define which strategy {{% vendor/name %}} uses to allocate resources
-when you first deploy your project or add a new container.
-The following strategies are available:
+## Set a resource initialization strategy
+
+{{% vendor/name %}} provides the following resource initialization strategies:
 
 | Strategy | Description |
 | ---------| ----------- |
-| `default`  | Initializes the new containers using the [{{% vendor/name %}} default resources](/manage-resources/resource-init.md). |
-| `manual`   | With this strategy, the first deployment fails and you need to configure resources manually through [the Console](/manage-resources/adjust-resources.md), or using `resources:set` in the CLI. |
-| `minimum`  | Initializes the new containers using the [{{% vendor/name %}} minimum resources](#minimum-resources). |
+| `default`  | Initializes the new containers using the [{{% vendor/name %}} default resources](#default-resources).</br>This strategy applies when you first deploy your project (or new container) unless you explicitly set another strategy, or allocate resources manually via the `resources:set` CLI command.  |
+| `manual`   | With this strategy, the first deployment fails and you need to configure resources manually through [the Console](/manage-resources/adjust-resources.md), or using `resources:set` in the CLI.</br></br> This strategy allows you to set the exact resources you want, with a single deployment. Other strategies may require fine-tuning, and therefore generate a second deployment. In this case, your environment would run for a short time with unwanted resources, and both deployments would generate downtime.|
```

Comment:
We don't support `manual` in case of backup restore either. It suffers from the same problem as having manual for branch.

Reply:
Manual has now been removed from env activate too.
I updated my comment above accordingly.

Reply:
Added a table at the end of the **Specify a resource initialization strategy** section + an info block at the beginning of every sub-section in case users go straight to one of them without reading the intro.